var/spack/repos/builtin/packages/openvslam/package.py

```diff
@@ -1,33 +0,0 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
-# Spack Project Developers. See the top-level COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-
-class Openvslam(CMakePackage):
```

Comment:
This package should be deprecated before being removed in case someone is still using it

Reply:
See https://github.com/spack/spack/pull/27374#discussion_r759480940

Reply:
By "remove" I mean deprecate first and delete in a later Spack release

Reply:
Even though there is no code, just a README?